blockchain/db/postgres/verification_db.py

```diff
@@ -120,20 +120,28 @@ def get_records(block_id, origin_id, phase):
         get_connection_pool().putconn(conn)
 
 
-def get_all(block_id, limit=None, offset=None, phase=None):
-    # Build query
+def get_all(limit=None, offset=None, **params):
+    """ return all verification records matching given parameters """
     query = SQL_GET_ALL
-    if block_id:
-        query += """ WHERE block_id = """ + str(block_id)
+    multi_param = False
+    if params:
+        query += """ WHERE """
 
-    if phase:
-        if not block_id:
-            query += """ WHERE """
-        else:
+    if "block_id" in params:
+        query += """ block_id = """ + str(params["block_id"])
+        multi_param = True
+
+    if "phase" in params:
+        if multi_param:
             query += """ AND """
-        query += """ phase = """ + str(phase)
+        query += """ phase = """ + str(params["phase"])
+        multi_param = True
 
-    query += """ ORDER BY block_id DESC """
+    if "origin_id" in params:
+        if multi_param:
+            query += """ AND """
+        query += """ origin_id = """ + str(params["origin_id"])
+        multi_param = True
```

Comment:
@j0j0r0 Added this in the event that we add more parameters later on. Want me to take it out?